praw/models/reddit/subreddit.py

```diff
@@ -404,6 +405,31 @@ def quaran(self):
         """
         return SubredditQuarantine(self)
 
+    @cachedproperty
+    def rules(self):
+        """Provide an instance of :class:`.SubredditRules`.
+
+        Use this attribute for interacting with a subreddit's removal reasons.
+        For example, to list all the removal reasons for a subreddit:
```

Comment:
Are removal reasons the same thing as rules?

Reply:
(It seems like they are, but it would be nice for the docstring to somehow subtly call this out)

Reply:
Ah crap, I made a typo while copy-pasting code. Actually, they are separate, but removal reasons are linked to rules.
```suggestion
        Use this attribute for interacting with a subreddit's rules.
        For example, to list all the rules for a subreddit:
```